packages/core/src/sync-store/encoding.ts

```diff
@@ -304,86 +304,12 @@ type RpcRequestResultsTable = {
   result: string;
 };
 
-type LogFiltersTable = {
-  id: string;
-  chainId: number;
-  address: Hex | null;
-  topic0: Hex | null;
-  topic1: Hex | null;
-  topic2: Hex | null;
-  topic3: Hex | null;
-  includeTransactionReceipts: 0 | 1;
-};
-
-type LogFilterIntervalsTable = {
-  id: Generated<number>;
-  logFilterId: string;
-  startBlock: ColumnType<string, string | bigint, string | bigint>;
-  endBlock: ColumnType<string, string | bigint, string | bigint>;
-};
-
-type FactoryLogFiltersTable = {
-  id: string;
-  chainId: number;
-  address: Hex;
-  eventSelector: Hex;
-  childAddressLocation: `topic${1 | 2 | 3}` | `offset${number}`;
-  topic0: Hex | null;
-  topic1: Hex | null;
-  topic2: Hex | null;
-  topic3: Hex | null;
-  includeTransactionReceipts: 0 | 1;
-};
-
-type FactoryLogFilterIntervalsTable = {
-  id: Generated<number>;
-  factoryId: string;
-  startBlock: ColumnType<string, string | bigint, string | bigint>;
-  endBlock: ColumnType<string, string | bigint, string | bigint>;
-};
-
-type TraceFiltersTable = {
-  id: string;
-  chainId: number;
-  fromAddress: Address | null;
-  toAddress: Address | null;
-};
-
-type TraceFilterIntervalsTable = {
-  id: Generated<number>;
-  traceFilterId: string;
-  startBlock: ColumnType<string, string | bigint, string | bigint>;
-  endBlock: ColumnType<string, string | bigint, string | bigint>;
-};
-
-type FactoryTraceFiltersTable = {
-  id: string;
-  chainId: number;
-  address: Hex;
-  eventSelector: Hex;
-  childAddressLocation: `topic${1 | 2 | 3}` | `offset${number}`;
-  fromAddress: Address | null;
-};
-
-type FactoryTraceFilterIntervalsTable = {
-  id: Generated<number>;
-  factoryId: string;
-  startBlock: ColumnType<string, string | bigint, string | bigint>;
-  endBlock: ColumnType<string, string | bigint, string | bigint>;
-};
-
-type BlockFiltersTable = {
-  id: string;
-  chainId: number;
-  interval: number;
-  offset: number;
-};
-
-type BlockFilterIntervalsTable = {
+type IntervalTable = {
   id: Generated<number>;
-  blockFilterId: string;
-  startBlock: ColumnType<string, string | bigint, string | bigint>;
-  endBlock: ColumnType<string, string | bigint, string | bigint>;
+  fragment_id: string;
+  chain_id: number;
+  start_block: ColumnType<string, string | bigint, string | bigint>;
+  end_block: ColumnType<string, string | bigint, string | bigint>;
```

Comment:
What do you think about migrating to multirange data types in the same PR/database migration? Should simplify this table a bit (can use fragment_id as PK), but comes with some risk/uncertainty. 

https://www.postgresql.org/docs/current/functions-range.html

Reply:
I think we are fine to ship without it. Specifically because numeric(78, 0) is not a supported range type, we would have to define our own.

 https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-DEFINING